tool.go

```diff
@@ -555,7 +571,7 @@ func (fs serveCommandFileSystem) Open(name string) (http.File, error) {
 	if isPkg || isMap || isIndex {
 		// If we're going to be serving our special files, make sure there's a Go command in this folder.
 		s := gbuild.NewSession(fs.options)
-		pkg, err := gbuild.Import(path.Dir(name[1:]), 0, s.InstallSuffix(), fs.options.BuildTags)
+		pkg, err := gbuild.Import(path.Dir(strings.Trim(name, "/")), 0, s.InstallSuffix(), fs.options.BuildTags)
```

Comment:
When the first character was unconditionally dropped with `name[1:]` it was always a slash. With this change the first character is sometimes a slash, but not always. I was seeing it pass strings like `ithub.com/jargv/...` into path.Dir, which fails, of course. `strings.Trim` seemed like a good way to conditionally drop the first character if it's a '/'. I'm new(ish) to go so I'd love to learn if there's a more idiomatic approach. :)